src/obs-vnc-source-thread.c

```diff
@@ -716,6 +724,15 @@ static void *thread_main(void *data)
 				SetDSCP(client->sock, client->QoS_DSCP = src->config.qosdscp);
 				src->dscp_updated = false;
 			}
+			if (src->skip_updated) {
+				pthread_mutex_lock(&src->config_mutex);
+				set_updateRect(src, client);
+				src->skip_updated = false;
+				pthread_mutex_unlock(&src->config_mutex);
```

Comment:
It's not consistent to lock `config_mutex` here but not locked at the other call of `set_updateRect`.